lib/capybara/cuprite/browser/page.rb

```diff
@@ -74,7 +74,7 @@ def visit(url)
         options = { url: url }
         options.merge!(referrer: referrer) if referrer
         response = command("Page.navigate", **options)
-        if response["errorText"] == "net::ERR_NAME_RESOLUTION_FAILED"
```

Comment:
I wonder if it's because of switching to chrome stable?

Reply:
That is an interesting point. On my development machine I'm using Chrome Canary, whereas Travis is using Stable. It seems to work on both for me. However I did wonder if it would be better to catch both responses 🤔

Reply:
Yea I think then we better catch both just